lib/package.gd

```diff
@@ -702,70 +702,80 @@ DeclareGlobalFunction( "LoadPackageDocumentation" );
 ##  true
 ##  ]]></Log>
 ##  <P/>
-##  The package name may be appropriately abbreviated. For example, at the time
-##  of writing, <C>LoadPackage("semi");</C> will load the 
-##  <Package>Semigroups</Package> package, and <C>LoadPackage("j");</C> will
-##  load the <Package>json</Package> package. If the abbreviation can not be
-##  uniquely completed, further suggestions will be offered.
-##  <P/>
-##  If the optional version string <A>version</A> is given,
-##  the package will only be loaded in a version number at least as large as
-##  <A>version</A>,
-##  or equal to <A>version</A> if its first character is <C>=</C>
+##  The package name is case insensitive and may be appropriately abbreviated. 
+##  At the time of writing, for example, <C>LoadPackage("semi");</C> 
+##  will load the <Package>Semigroups</Package> package, and 
+##  <C>LoadPackage("j");</C> will load the <Package>json</Package> package. 
+##  If the abbreviation cannot be uniquely completed, 
+##  a list of available completions will be offered, 
+##  and <Ref Func="LoadPackage"/> returns <K>fail</K>.
+##  Thus the names of <E>all</E> installed packages can be shown by calling
+##  <C>LoadPackage("");</C>.
+##  <P/>
+##  When the optional argument string <A>version</A> is present,
+##  the package will only be loaded in a version number 
+##  equal to or greater than <A>version</A> 
 ##  (see&nbsp;<Ref Func="CompareVersionNumbers"/>).
-##  The argument <A>name</A> is case insensitive.
+##  If the first character is <C>=</C> then only that version will be loaded. 
```

Comment:
... first character of <A>version</A> is ...

Reply:
Thanks for your comments - I now need some git advice. 
Before doing any of the suggested editing I did the following: 
(1) in master, did: git pull upstream master, then git push origin 
(2) git checkout manual76again, then git rebase master 
(3) attempted git push origin manual76again , but got error message: 
"error: failed to push some refs to 'https://github.com/cdwensley/gap'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart." etc. 
Looks like I've made another mess of things?  Any suggestions?

Reply:
Whenever you perform a rebase, you will have to add `--force` to the git push after that.

In general my advice would be to not worry about rebasing and merging master in -- unless your commit interferes with anyone else, it's fine to drift behind slightly.